cadquery/assembly.py

```diff
@@ -106,16 +107,31 @@ def _getAxis(self, arg: Shape) -> Vector:
 
         return rv
 
-    def _getPlane(self, arg: Shape) -> Plane:
+    def _getPln(self, arg: Shape) -> gp_Pln:
 
         if isinstance(arg, Face):
-            normal = arg.normalAt()
+            rv = arg.toPln()
```

Comment:
To my understanding the center of the underlying geometry does not need to correspond to the center of the trimmed face. Should we somehow handle this here?

Reply:
Absolutely, that was my mistake. I think I can re-use `self._getPnt` for this, check out the latest commit.